lib/hooks/kibela/hook.rb

```diff
@@ -0,0 +1,24 @@
+module Idobata::Hook
+  class Kibela < Base
+    screen_name 'kibe.la'
+    icon_url hook_image_url('icon.png')
```

Comment:
It seems there is not the `icon.png` here but `icon.jpg` is. I'd like to use png format rather than jpg as far as possible. How about replacing `icon.jpg` with  this <img alt='' src='https://cdn-ak.f.st-hatena.com/images/fotolife/d/dex1t/20161025/20161025012831.png' width=20 height=20 /> ? (It can be found in [this article](http://blog.kibe.la/entry/2016/11/01/100000)).